agent/client/channel/channel_test.go

```diff
@@ -104,55 +104,56 @@ func setup(t *testing.T, connect func(agentpb.Agent_ConnectServer) error, expect
 	return
 }
 
-func TestAgentRequestWithTruncatedInvalidUTF8(t *testing.T) {
-	defaultMaxQueryLength := truncate.GetDefaultMaxQueryLength()
-	fingerprint, _ := truncate.Query("SELECT * FROM contacts t0 WHERE t0.person_id = '?';", defaultMaxQueryLength)
-	invalidQuery := "SELECT * FROM contacts t0 WHERE t0.person_id = '\u0241\xff\\uD83D\xddÃ¼\xf1'"
-	query, _ := truncate.Query(invalidQuery, defaultMaxQueryLength)
+/*
```

Comment:
Do we intend to revisit the ones that have been commented out one day?

Reply:
good point will revert the changes to make sure that everything is fine